### ISSUE TITLE
is there any function in Client part like "On (String, Callback)"? 

### ISSUE THREAD

**Issue by Indigodevops20** [Textalk/websocket-php#134]
well I've been testing and I really like this library, the only problem is that I don't see a function as it is and I would like to know if there will be in the future

**Comment by sirn-se** [comment]
Yes, the plan is to implement a listener function. But it require a major refactoring of connection handling, so it might take some time until we get there.
